unlock-app/src/services/web3Service.js

```diff
@@ -680,9 +680,13 @@ export default class Web3Service extends EventEmitter {
             }
           )
         })
+        .catch(() => {
+          // Triggered when there are missing keys
+          return null
```

Comment:
is there a way to log this error, just in case something more serious happens?

Reply:
I think this is actually not an error per se. This is hos solidity tells us that there is no such key at that address.